osu/map.py

```diff
@@ -52,9 +53,26 @@ def files(self):
 
     def zip(self):
         map_pack_file = MAP_PACKS_FOLDER.joinpath(f"{uuid.uuid4()}.zip")
-
+        print(map_pack_file)
         files = self.files()
         FileTools.zip_files(files, to_file=map_pack_file)
-        FileTools.delete_files(files)
 
         return map_pack_file
+
+    def upload(self, dbx_path: str) -> str | None:
```

Comment:
why do we need to expose the dbx path to the user? 

we can just define it in the upload function based off what zip returns (if you need it as a string, do str(path))

Reply:
again, a true tipsy coding moment, change committed